### PR TITLE
Implement Leech Seed option 

### PR DESCRIPTION
**Pull request by AustinXII**

### CODE REVIEW

File: calc_bc.html [hunk 1]
```diff
@@ -584,6 +584,14 @@
                             <input class="btn-input" type="checkbox" id="lightScreenR" />
                             <label class="btn btn-xwide btn-right" for="lightScreenR">Light Screen</label>
                         </div>
+                        <div class="left" title="Has this Pok&eacute;mon been effected by Leech Seed?">
```
Comment (Zarel) [inline calc_bc.html]: "affected"

Comment (AustinXII) [inline calc_bc.html]: oops

**Comment by Marty-D** [comment]
A nice touch could be if setting Liquid Ooze on the Leech Seeded side would subtract that Pokemon's maxHP/8 from calcs against the opposite side. Something to consider!

**Comment by AustinXII** [comment]
@Marty-D will do, there's probably a few additions I could do that involves Liquid Ooze

**Pull request merged**